whois-parser.gemspec

```diff
@@ -11,7 +11,288 @@ Gem::Specification.new do |s|
+  s.files = [
+    ".yardopts".freeze,
```

Comment:
@raszi the gemspec is generated automatically, any change here will be overridden.